README.md

```diff
@@ -15,20 +15,27 @@ For more information, please refers to [BUILDING.md](BUILDING.md).
 If you'd like to contribute, please review the
 [Contributing Guidelines](CONTRIBUTING.md).
 
-## End-to-End Testing
+## Testing
 ### Requirements
 
 - [Python3.6+](https://www.python.org/)
 - [tox](https://pypi.org/project/tox)
 - [Vagrant](https://www.vagrantup.com/)
 - [VirtualBox](https://www.virtualbox.org)
 
-### Usage
+### Bootstrapping local environment
 
-To run tests locally:
-```shell
+```shell 
+# Install virtualbox guest addition plugin
+vagrant plugin install vagrant-vbguest
 # Bootstrap a platform on a vagrant environment using
-./doit.sh vagrantup
+./doit.sh vagrant_up
+```
+
+### End-to-End Testing
+
+To run tests locally: Complete the bootstrapping step above
```

Comment:
`To run the test-suite locally, first complete the bootstrap step as outline above.`